dataland-frontend/src/components/generics/ViewMultipleDatasetsDisplayBase.vue

```diff
@@ -26,6 +26,14 @@
               :companyId="companyId"
               :singleDataMetaInfoToDisplay="singleDataMetaInfoToDisplay"
             />
+            <MultiLayerDataTableFrameworkPanel
+              v-if="dataType === DataTypeEnum.EutaxonomyFinancials"
```

Comment:
pushed

Reply:
Can we change it in a separate task? Because it concerns EutaxonomyFinancial and EutaxonomyNonFinancial on frontend and on backend and it will infect lots of field